pom.xml

```diff
@@ -19,10 +19,48 @@
     <url>https://github.com/eclipse-ee4j/cargotracker/issues</url>
   </issueManagement>
   <properties>
+    <!-- TODO Moving to 1.8.1.Final+ causes glassfish profile to fail -->
```

Comment:
Are we being consistent about using Maven properties? I typically only use it for very prominent dependencies like Jakarta EE or if something is used multiple times. Otherwise I just leave it embedded (I actually find it easier to maintain that way). Any sensible scheme is fine, as long as we are being conscious and consistent. Can you please give it some more thought and adjust accordingly?

Reply:
I've updated this section and added a comment as to how the properties are being used and made the rest consistent.  LMK if you want a different approach though!